governance.Jenkinsfile

```diff
@@ -20,8 +23,17 @@ podTemplate(label: label) {
                             checkout scm
                         }
 
-                        stage("validate external links") {
-                            validateLinks('find . -name "*.md" | grep -v "vendor" | grep -v "./call-ec/web-ui/README.md"')
+                        // Enable on PR when internal links will be fixed
```

Comment:
I didn't quite get this. If it is not working in PRs right now, why do we want to have it?

Reply:
If fix for not working internal links will be merged before this I will enable it. If not it will be disabled until links will be fixed to not block PR on this repository, because of not working old links.